web/src/beta/features/PluginPlayground/Plugins/index.tsx

```
@@ -35,27 +36,35 @@ const Plugins: FC<Props> = ({
-              key={plugin.id}
-              selected={selectedPlugin.id === plugin.id}
-              onClick={() => selectPlugin(plugin.id)}
-            >
-              {plugin.title}
-            </PluginListItem>
-          ))}
-        </PluginList>
-      </PluginListWrapper>
+      <PluginList>
+        {presetPlugins.map((category) => (
+          <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)